pyuvdata/uvdata/tests/test_uvh5.py

```diff
@@ -62,6 +62,24 @@ def uv_partial_write(casa_uvfits, tmp_path):
     return
 
 
+@pytest.fixture(scope="session")
+def sma_mir_main():
+    # read in test file for the resampling in time functions
+    uv_object = UVData()
+    testfile = os.path.join(DATA_PATH, "sma_test.mir")
+    uv_object.read(testfile)
+
+    yield uv_object
+
+
+@pytest.fixture(scope="function")
+def sma_mir(sma_mir_main):
+    # read in test file for the resampling in time functions
+    uv_object = sma_mir_main.copy()
+
+    yield uv_object
+
+
```

Comment:
I don't see where these fixtures are used...

Reply:
Whoops -- they _were_ used, but I moved a test out of `test_uvh5` and into `test_mir` instead. Fixed in a450660.